### ISSUE TITLE
Can not login 

### ISSUE THREAD

**Issue by victorcpereira** [signalpoint/jDrupal#38]
I'm trying to make login using JDrupal, but I can't.
I followed all instructions on readme file to install the module.
I can get node data. But I cannot login 

I'm using the following code written in coffeeScript:

`
 jDrupal.userLogin("user", "pass").then (data) ->
      console.log jDrupal.currentUser().id()
`

But always return an anonymous user.
Can you help me what I'm missing?

**Comment by signalpoint** [comment]
@victorcpereira Sounds like you're having cross site request issues. Please elaborate more on your environments, i.e. where is drupal, where is your app, is it a compiled app, a web app, a headless app, what are the URL(s) involved, etc.

**Comment by victorcpereira** [comment]
Sorry, I posted the question before finish it unintentionally.

I'm developing an Ionic App that gets data from Drupal.
I can get Nodes and Users without issues.

But I cannot login. It always return an anonymous user.

**Comment by signalpoint** [comment]
@victorcpereira no worries. I personally have never built an Ionic App so I'm not sure what's going on there. I'd imagine it has something to do with the cookie/session/session-id not being set properly on the device. PhoneGap seems to have always taken care of that for me (or all modern browsers if running a web app), but I can't speak to what an Ionic environment is like. But I would recommend starting to investigate cookies/sessions/session-ids on the device before, during, and after login attempts.

**Comment by victorcpereira** [comment]
@signalpoint I checked out the response from request and it is returning the HTML from login page.
Have you seen something like that?

**Comment by kentr** [comment]
@victorcpereira maybe this will provide some clues: http://blog.ionic.io/angularjs-authentication/

**Comment by signalpoint** [comment]
@victorcpereira the html response is normal for now, because that's the way a .js user login works in D8 (for now I guess): https://www.drupal.org/documentation/modules/rest/javascript

**Comment by victorcpereira** [comment]
To feedback the question,
The issue was a simple slash.

In Jdrupal.settings I set the "sitePath" as http://example.com/ (with the slash in the end) and "basePath" as just "/". 

Like this, the request went to http://example.com//user/login (with two slashes before user), then it redirects to http://www.example.com/user/login without any parameter.

I've just removed the slash in the end of "sitePath" and it's working fine.

Thanks for the help.
Regards!
